kubernetes/monitoring/services/cert-checker.yaml

```diff
@@ -0,0 +1,103 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: cert-checker
+  namespace: http
+spec:
+  type: ClusterIP
+  ports:
+    - port: 8888
+      protocol: TCP
+      name: http
+  selector:
+    app: cert-checker
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cert-checker
+  namespace: http
+  labels:
+    app: cert-checker
+data:
+  configs.json: |
+    [
+      {"cert": "/etc/secrets/robotcert.pem", "ckey": "/etc/secrets/robotkey.pem", "keytab": ""},
+      {"cert": "/etc/secrets/server.pem", "ckey": "/etc/secrets/server-key.pem", "keytab": ""},
+      {"cert": "/etc/secrets/robot-training-cert.pem", "ckey": "/etc/secrets/robot-training-key.pem", "keytab": ""},
+      {"cert": "/etc/secrets/tls.crt", "ckey": "/etc/secrets/tls.key", "keytab": ""},
+      {"cert": "/etc/secrets/tls.crt", "ckey": "/etc/secrets/tls.key", "keytab": ""},
+      {"cert": "", "ckey": "", "keytab": "/etc/secrets/cmsmonit_keytab"},
+      {"cert": "", "ckey": "", "keytab": "/etc/secrets/cmspopdb_keytab"},
+      {"cert": "", "ckey": "", "keytab": "/etc/secrets/sqoop_keytab"}
+    ]
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: cert-checker
+  name: cert-checker
+  namespace: http
+spec:
+  selector:
+    matchLabels:
+      app: cert-checker
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cert-checker
+      annotations:
+        prometheus.io/scrape: 'true'
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "8888"
+    spec:
+      containers:
+        - image: registry.cern.ch/cmsmonitoring/cert-checker:test
```

Comment:
please do not use concrete image tag and rather replace `:test` with ` #imagetag` which we use in `deploy-srv.sh` script when we deploy services